x-pack/plugins/security_solution/common/endpoint/data_generators/exceptions_list_item_generator.ts

```diff
@@ -29,7 +29,7 @@ export class ExceptionsListItemGenerator extends BaseDataGenerator<ExceptionList
           field: ConditionEntryField.PATH,
           operator: 'included',
           type: 'match',
-          value: '/one/two/three',
+          value: 'c:\\fol\\bin.exe',
```

Comment:
Why change this to Windows?

Maybe randomize it? like:

```suggestion
          value: this.randomChoice(['/one/two/three', 'c:\\one\\two\\three']
```

(maybe create a `const` for the array). Of you could at the OS and add the appropriate path value based on that.

Reply:
Yeah, upon looking at the tests that use this for mocked data, I found that they verify Windows properties for TAs and some of them had non-windows paths. So made this change to reflect that accurately. 

Randomizing this would break the tests though. I'll add a condition here instead to use os value instead.

Reply:
done c7a17f43c3d8a333289f39b86926398c53996418